docker-compose.yml

```diff
@@ -27,4 +31,21 @@ web:
   ports:
     - "8077:8077"
   links:
-    - db
\ No newline at end of file
+    - db
+celery:
+  image: ccxcon_web
+  mem_limit: 384m
```

Comment:
You can get rid of this line, it was an attempt at restricting memory usage in LORE to diagnose our frequent out of memory errors. It's only meant for development use, but this will run in the test environment